dll/shellext/fontext/CEnumFonts.cpp

```diff
@@ -21,20 +19,20 @@ class CEnumFonts :
 
 public:
     CEnumFonts()
-        :m_dwFlags(0)
-        ,m_Index(0)
+        : m_dwFlags(0)
+        , m_Index(0)
     {
     }
 
-    STDMETHODIMP Initialize(CFontExt* folder, DWORD flags)
+    HRESULT Initialize(CFontExt* folder, DWORD flags)
```

Comment:
Why remove `STDMETHODIMP` ?
`override` is missing here btw

Reply:
`override` causes error.
![image](https://github.com/user-attachments/assets/a72ebfc2-4a22-4109-aa24-32df880b29b9)

What interface?

Reply:
sorry, this is not a COM function at all.

Reply:
ship it I guess.